.ebextensions/env-file-creation.config

```
@@ -25,17 +26,27 @@ files:
         else
             echo "Directory ${TARGET_DIR} already exists!"
         fi
-        echo "Creating config for ${ENV_NAME} in ${AWS_REGION}"
-        aws ssm get-parameter --name "${ENV_NAME}-general" --with-decryption --region $AWS_REGION | jq -r '.Parameter.Value' > $TARGET_DIR/.env
-        aws ssm get-parameter --name "${ENV_NAME}-captcha" --with-decryption --region $AWS_REGION | jq -r '.Parameter.Value' >> $TARGET_DIR/.env
-        aws ssm get-parameter --name "${ENV_NAME}-ga" --with-decryption --region $AWS_REGION | jq -r '.Parameter.Value' >> $TARGET_DIR/.env
-        aws ssm get-parameter --name "${ENV_NAME}-intranet" --with-decryption --region $AWS_REGION | jq -r '.Parameter.Value' >> $TARGET_DIR/.env
-        aws ssm get-parameter --name "${ENV_NAME}-sentry" --with-decryption --region $AWS_REGION | jq -r '.Parameter.Value' >> $TARGET_DIR/.env
-        aws ssm get-parameter --name "${ENV_NAME}-sms" --with-decryption --region $AWS_REGION | jq -r '.Parameter.Value' >> $TARGET_DIR/.env
-        aws ssm get-parameter --name "${ENV_NAME}-ndi" --with-decryption --region $AWS_REGION | jq -r '.Parameter.Value' >> $TARGET_DIR/.env
-        aws ssm get-parameter --name "${ENV_NAME}-sgid" --with-decryption --region $AWS_REGION | jq -r '.Parameter.Value' >> $TARGET_DIR/.env
-        aws ssm get-parameter --name "${ENV_NAME}-verified-fields" --with-decryption --region $AWS_REGION | jq -r '.Parameter.Value' >> $TARGET_DIR/.env
-        aws ssm get-parameter --name "${ENV_NAME}-webhook-verified-content" --with-decryption --region $AWS_REGION | jq -r '.Parameter.Value' >> $TARGET_DIR/.env
+
+        echo "Creating ENV_NAME config..."
+        if [ -n "${ACTUAL_ENV_NAME}" ]; then
+            echo "setting ENV_NAME as ACTUAL_ENV_NAME"
```

```suggestion
            echo "setting ENV_NAME as ACTUAL_ENV_NAME (${ACTUAL_ENV_NAME})"
```
Maybe we can be more verbose and `echo` the actual env var to be loaded as well to it'll be easier for us to debug in the future.

---

makes sense! done in [fe6420c](https://github.com/opengovsg/FormSG/pull/6181/commits/fe6420c5f73b85a4ac72574513819c24cded4d8d)